core/object.cpp

```diff
@@ -1686,7 +1686,7 @@ void Object::_bind_methods() {
 		mi.name = "emit_signal";
 		mi.arguments.push_back(PropertyInfo(Variant::STRING, "signal"));
 
-		ClassDB::bind_vararg_method(METHOD_FLAGS_DEFAULT, "emit_signal", &Object::_emit_signal, mi);
+		ClassDB::bind_vararg_method(METHOD_FLAGS_DEFAULT, "emit_signal", &Object::_emit_signal, mi, Vector<Variant>(), false);
```

Comment:
Other similar methods seem to use `varray()` as a shorthand for `Vector<Variant>()`.